### DISCUSSION TITLE
Bloom Effect

### DISCUSSION THREAD

**Discussion by tjzzhang** [bnpr/Malt#428] [Q&A] [answered]
![bloom](https://user-images.githubusercontent.com/9721816/197796391-23504b49-a877-4151-b5ba-f34e4e8c39e8.png)

I have learned from the the bloom effect example, the bloom effect was set in screen script, it seems like a global setting, works for all the objects in the scene. How can I make some object have bloom effect, some didn't?

**Comment by pragma37** [comment, marked as answer]
https://github.com/bnpr/Malt/discussions/347#discussioncomment-3109901 👍
